_episodes/03-high-dim-modeling.md

```diff
@@ -4,43 +4,121 @@ teaching: 0
+1. Ames house prices dataset (sklearn: https://scikit-learn.org/stable/modules/generated/sklearn.datasets.load_boston.html)
+   * 1460 observations, 80 feautres (bingo!)
+2. Boston house prices dataset (sklearn)
+   * 506 observations, 14 different features; easy to fit regression model to
+   * Evidently there's an ethical issue with this dataset. Recommended to not use (https://scikit-learn.org/stable/modules/generated/sklearn.datasets.load_boston.html).
+3. Diabetes
+   * 442 observations, 10 features, easy to fit regression model to
+
+# Functions Provided
+1. Load/plot data
 
 # Introduction
-- Curse of dimensionality
-- Bias VS Variance, 10X as many datapoints as features as general rule of thumb
-- Methods for feature selection
+## Machine learning — a brief overview
+Machine learning is the process of learning (optimizing) a function that is able to map some input to a desired output.
 
-> ## Exercise - demonstrate overfitting (might just show this in intro, but leaving here as placeholder for now)
-> 
-> 
->
-> > ## Solution
-> >  
-> >  
-> {: .solution}
-{: .challenge}
+**Find/make diagram**: input data --> cool algorithms --> prediction forecast, recommendation ,decision
 
-> ## Exercise
-> In this example, we would like to classify images of cats versus dogs. In every image example, a cat or a dog appears at the center of the image with some background imagery present as well. There are two example images provided below. Instead of training our model on every pixel present in each image, what could we do to help the model hone in on the important aspects of the images that relate to how dogs and cats differ?
-> 
->
-> > ## Solution
-> >  - Include only the center of each image--where a dog or a cat appears
-> >  - Include only pixels that contain the head of the animal--where differences are more noticeable between the species.
-> {: .solution}
-{: .challenge}
+The techniques breakdown into two broad categories, predictors and classifiers. Predictors are used to predict a value (or set of value) given a set of inputs, for example trying to predict the cost of something given the economic conditions and the cost of raw materials or predicting a country’s GDP given its life expectancy. Classifiers try to classify data into different categories, for example deciding what characters are visible in a picture of some writing or if a message is spam or not. Some examples might include:
+* predicting a person’s weight based on their height
+* predicting commute times given traffic conditions
+* predicting house prices given stock market prices
+* classifying if an email is spam or not
+* classifying what if an image contains a person or not
+
+In this lesson, we will work with predictive models in both a low and high-dimensional data regime.
+
+## Training data 
+* The text below is borrowed form this lesson: https://carpentries-incubator.github.io/machine-learning-novice-sklearn/01-introduction/index.html)
```

Comment:
```suggestion
* The text below is borrowed form this lesson the [Carpentries Machine Learning with Sklearn Incubator Lesson](https://carpentries-incubator.github.io/machine-learning-novice-sklearn/01-introduction/index.html)
```
Was missing the syntax for a link and think a descriptive link would be good.

Reply:
How much of the text below is from the ML for Sklearn lesson? Maybe putting it in a block quote would be good?

> ML using slkearn lesson quote

Reply:
We also want to be careful with out much we use since we will have to maintain it as their lesson develops